x-pack/platform/plugins/shared/stack_connectors/public/connector_types/openai/openai.tsx

```diff
@@ -11,6 +11,8 @@ import type { GenericValidationResult } from '@kbn/triggers-actions-ui-plugin/pu
 import { SUB_ACTION } from '../../../common/openai/constants';
 import { OPENAI_CONNECTOR_ID, OPENAI_TITLE } from '../../../common/openai/constants';
 import type { ActionParams, OpenAIConnector } from './types';
+import { formSerializer } from './form_serialization';
+import { formDeserializer } from '../lib/webhook/form_serialization';
```

Comment:
Let's duplicate this even if it is the same. It can diverge in the future, and we have to remember that this connector uses it.